dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/WsPingPongBaseSuite.scala

```diff
@@ -136,7 +136,10 @@ abstract class WsPingPongBaseSuite extends WsSuiteBase {
       }
 
       val expectedError = InvalidJson(Nil)
-      val List(actualInvalidJsonError, actualPongTimeoutError) = errors
+      val (actualInvalidJsonError, actualPongTimeoutError) = errors match {
+        case List(actualInvalidJsonError, actualPongTimeoutError) => (actualInvalidJsonError, actualPongTimeoutError)
+        case _ => throw new IllegalArgumentException(s"Unexpected error")
```

Comment:
Same. Also there is a simple rule for exceptions I always use:
1. If we are inside a function and we got an invalid argument, we throw `IllegalArgumentException` (if throwing is acceptable)
2. If we outside the function is should be another exception type, e.g. `RuntimeException` which is totally fine here

Reply:
ok, got it